packages/runtime-dom/src/apiCustomElement.ts

```diff
@@ -111,37 +123,45 @@ export function defineCustomElement<
     Extends,
     E,
     EE
-  > & { styles?: string[] }
+  > & { styles?: string[] },
+  config?: DefineCustomElementConfig
 ): VueElementConstructor<ExtractPropTypes<PropsOptions>>
 
 // overload 5: defining a custom element from the returned value of
 // `defineComponent`
-export function defineCustomElement(options: {
-  new (...args: any[]): ComponentPublicInstance
-}): VueElementConstructor
+export function defineCustomElement(
+  options: {
+    new (...args: any[]): ComponentPublicInstance
+  },
+  config?: DefineCustomElementConfig
+): VueElementConstructor
 
 export function defineCustomElement(
   options: any,
-  hydate?: RootHydrateFunction
+  config?: DefineCustomElementConfig,
+  hydrate?: RootHydrateFunction
 ): VueElementConstructor {
   const Comp = defineComponent(options as any)
   class VueCustomElement extends VueElement {
     static def = Comp
     constructor(initialProps?: Record<string, any>) {
-      super(Comp, initialProps, hydate)
+      super(Comp, initialProps, config, hydrate)
     }
   }
 
   return VueCustomElement
 }
 
-export const defineSSRCustomElement = ((options: any) => {
-  // @ts-ignore
-  return defineCustomElement(options, hydrate)
+export const defineSSRCustomElement = ((
+  options: any,
+  config?: DefineCustomElementConfig
+) => {
+  // @ts-expect-error
```

Comment:
I think this was accidentally replaced.

```suggestion
  // @ts-ignore
```